### DISCUSSION TITLE
Inserção do logo do Vivo Empresas no componente MainNavigationBar / FunnelNavigationBar

### DISCUSSION THREAD

**Discussion by juliaisaway** [Telefonica/mistica-design#1877] [Proposals]
### Goal

Trazer a discussão sobre a possibilidade de ter uma variação de logo para o Vivo Empresas, uma marca alternativa da Vivo

### Context

Hoje dentro do Vivo Empresas, passamos a usar o Mística, porém o logo atualmente é um campo fixo que não conseguimos alterar com facilidade. Como o logo do Vivo Empresas tem dimensões diferentes, seria bom poder adaptar melhor ao contexto.

### Impact

Por conta dessa divergência de logo, o time do Vivo Empresas entende que é necessário fazer um outro componente de navigation bar separado, com o logo correto do produto, o que acredito ser uma má prática do lado de DS

### Discussion Questions

Podemos incluir como uma das variáveis dentro de Brand Logo no componente?

### Do you have a visual proposal?

https://www.figma.com/design/pagjUAkxWQUzIGkahywRg8/Vivo-Empresas---Aplica%C3%A7%C3%A3o-de-Logo?node-id=0-1

**Comment by yceballost** [comment]
Hola Julia!

Veo dos cosas aquí.

1. La posibilidad de usar otro logotipo en el componentes NavigationBar en Figma
2. Discutir sobre la idea de incluir otros logotipos o submarcas de Vivo dentro del componente Logo

## 1. Incluir logotipos custom en la navigation bar de Figma
Esto es algo que a día de hoy ya es posible, solo tienes que tener el logotipo que quieras usar en un componente de Figma y reemplazar el componente Logo de la navigation bar por el que has creado.

Ejemplo:
https://www.figma.com/design/c15cHpAU7t3VRE4UFc62yH/Custom-Logo-in-Navigation-Bar?node-id=0-1&t=HeXdfiwow0T0FFIa-1

## 2. Incluir submarcas dentro del componente Logo
En principio creemos que puede ser dificil de escalar el componente si incluimos todas las posibles submarcas que podrían existir en todas las marcas.

Para esto creemos que quizás lo que tendría sentido es que esto lo gestionéis con un extended donde vosotros seáis los que controla la cantidad de submarcas que puedan existir dentro de Vivo.

Entendemos que estas submarcas tendrían un uso mucho más reducido en los productos digitales como para mantenerlo en la librería global y por eso creemos que puede ser mejor tenerlo en la extended.

De todas formas, si se te ocurre alguna forma de que esto pudiera llegar a ser escalable, sientete libre de compartirnos la idea :)

Se me ocurre que quizás estos logos de submarcas podrían ir en esa posible librería de Assets que estuvimos hablando con @vinrosa
